few errors while building 0.13 debian pkg

lazy me decided just to ask how to fix/address those two 

```
======================================================================
ERROR: Test CombineMarkersModel Traits Model
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/python-mne-0.13+dfsg/mne/utils.py", line 827, in dec
    return function(*args, **kwargs)
  File "/build/python-mne-0.13+dfsg/mne/gui/tests/test_marker_gui.py", line 33, in test_combine_markers_model
    assert_false(model.mrk3.can_save)
AttributeError: 'function' object has no attribute 'can_save'

======================================================================
FAIL: Test CombineMarkersModel Traits Model.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib/python2.7/dist-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/build/python-mne-0.13+dfsg/mne/utils.py", line 827, in dec
    return function(*args, **kwargs)
  File "/build/python-mne-0.13+dfsg/mne/gui/tests/test_kit2fiff_gui.py", line 35, in test_kit2fiff_model
    assert_false(model.can_save)
AssertionError: <bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7ff3e84eea10>> is not false
    '<bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7ff3e84eea10>> is not false' = self._formatMessage('<bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7ff3e84eea10>> is not false', "%s is not false" % safe_repr(expr))
>>  raise self.failureException('<bound method Kit2FiffModel.<lambda> of <mne.gui._kit2fiff_gui.Kit2FiffModel object at 0x7ff3e84eea10>> is not false')
```

--- a/mne/gui/tests/test_kit2fiff_gui.py
+++ b/mne/gui/tests/test_kit2fiff_gui.py
@@ -13,6 +13,7 @@ import mne
 from mne.io.kit.tests import data_dir as kit_data_dir
 from mne.io import read_raw_fif
 from mne.utils import _TempDir, requires_traits, run_tests_if_main
+from mne.utils import requires_mayavi
 
 mrk_pre_path = os.path.join(kit_data_dir, 'test_mrk_pre.sqd')
 mrk_post_path = os.path.join(kit_data_dir, 'test_mrk_post.sqd')
@@ -25,6 +26,7 @@ warnings.simplefilter('always')
 
 
 @requires_traits
+@requires_mayavi
 def test_kit2fiff_model():
     """Test CombineMarkersModel Traits Model."""
     from mne.gui._kit2fiff_gui import Kit2FiffModel, Kit2FiffPanel

--- a/mne/gui/tests/test_marker_gui.py
+++ b/mne/gui/tests/test_marker_gui.py
@@ -12,6 +12,7 @@ from nose.tools import assert_true, assert_false
 from mne.io.kit.tests import data_dir as kit_data_dir
 from mne.io.kit import read_mrk
 from mne.utils import _TempDir, requires_traits, run_tests_if_main
+from mne.utils import requires_mayavi
 
 mrk_pre_path = os.path.join(kit_data_dir, 'test_mrk_pre.sqd')
 mrk_post_path = os.path.join(kit_data_dir, 'test_mrk_post.sqd')
@@ -21,6 +22,7 @@ warnings.simplefilter('always')
 
 
 @requires_traits
+@requires_mayavi
 def test_combine_markers_model():
     """Test CombineMarkersModel Traits Model"""
     from mne.gui._marker_gui import CombineMarkersModel, CombineMarkersPanel

--- a/mne/utils.py
+++ b/mne/utils.py
@@ -796,14 +796,11 @@ def requires_module(function, name, call=None):
 
     @wraps(function)
     def dec(*args, **kwargs):
-        skip = False
         try:
             exec(call) in globals(), locals()
-        except Exception:
-            skip = True
-        if skip is True:
-            raise SkipTest('Test %s skipped, requires %s'
-                           % (function.__name__, name))
+        except Exception as exc:
+            raise SkipTest('Test %s skipped, requires %s. Got exception (%s)'
+                           % (function.__name__, name, exc))
         return function(*args, **kwargs)
     return dec
 

